### ISSUE TITLE
[Idea] Add equitable content resources to site

### ISSUE THREAD

**Issue by theecrit** [theecrit/contentandux#32]
**What specific challenge does your idea address?**
In order to view the existing [list of resources](https://bit.ly/support-black-content-design-strategists), I need to know the dedicated URL or search the Slack group to find the file. This limits the accessibility and visibility of the material.

**Describe the solution you'd like**
Maintain this info as a dedicated page on the site. This could be an embedded file (the goal of keeping it editable for members should be preserved if possible), or even an AirTable database (more effort, different UX impact), or just a static page updated via git.

**What other approaches have you considered?**
Leaving things as-is.

**Additional context or details**
- [Public Slack file of resources](https://bit.ly/support-black-content-design-strategists)
